### ISSUE TITLE
(Request) Tutorial reading images directory

### ISSUE THREAD

**Issue by Rverm** [GetmeUK/ContentTools#227]
Hi

Any chance you could make a (simple) tutorial on how to read an image directory and make an image clickable instead of uploading them? So an "Insert image" button which just shows all images which are clickable for inserting? 

Thanks!

**Comment by anthonyjb** [comment]
Hi @Rverm are you looking to build the media manager yourself or are you thinking about integrating an external media manager (such as KCFInder). The reason I ask is that @concept-core is currently looking at creating a tutorial to cover KCFinder integration which supports selecting from an existing folder/list of images.

**Comment by concept-core** [comment]
@Rverm Jup that is completely true.
If you use php I will be able to help you with what you need, but I do use KCFinder. ( keep that in mind )

**Comment by Rverm** [comment]
@anthonyjb I would prefer building my own media manager, but I've looked at KCFinder and it also looks cool. 

@concept-core That's exactly what I'm looking for! Looking forward to it!

**Comment by concept-core** [comment]
@Rverm I Will try to create the tutorial this weekend.
There is only one thing I need to know for you specific.

Do you use a framework by any change ? 
If so which one.

**Comment by Rverm** [comment]
@concept-core I'm using jQuery, and Code Igniter for the php part. Thanks again!

**Comment by concept-core** [comment]
@Rverm the newer version of codeigniter? 
Or the old one when hosted by Ellis lab ?

**Comment by Rverm** [comment]
@concept-core the latest version, 3.0.6 :)

**Comment by concept-core** [comment]
@Rverm i am going to Wright the tutorial Today.
If you wish you can message me.

**Comment by zaka786khan** [comment]
yes i want to make a tutorial on how to integrate KCFinder inside contenttools. It will really be appreciable and make contenttools more usable and effectable.

regards.
